EmailValidator/Validation/EmailValidation.php

```diff
@@ -21,6 +21,8 @@ public function isValid($email, EmailLexer $emailLexer);
     /**
      * Returns the validation error.
      *
+     * Be careful about no-error does not necessarily mean email was valid.
```

Comment:
I'm assuming `NoRFCWarningsValidation` seems to be on wrong side. 
So I will concern about this in another PR. (I'm gonna remove above line here.)